examples/generalized_linear_models/GLM-ordinal-regression.myst.md

```diff
@@ -20,6 +20,10 @@ kernelspec:
 :::
 
 ```{code-cell} ipython3
+import warnings
+
+warnings.simplefilter(action="ignore", category=FutureWarning)
```

Comment:
We should aim to fix the warnings instead of ignoring them. Can you share what warnings you got? We might be able to help

Reply:
I got:
```
[/Users/mihagazvoda/miniconda3/envs/pymc-dev/lib/python3.11/site-packages/arviz/stats/stats.py:307](https://file+.vscode-resource.vscode-cdn.net/Users/mihagazvoda/miniconda3/envs/pymc-dev/lib/python3.11/site-packages/arviz/stats/stats.py:307): FutureWarning: Setting an item of incompatible dtype is deprecated and will raise in a future error of pandas. Value 'False' has dtype incompatible with float64, please explicitly cast to a compatible dtype first.
  df_comp.loc[val] = (
[/Users/mihagazvoda/miniconda3/envs/pymc-dev/lib/python3.11/site-packages/arviz/stats/stats.py:307](https://file+.vscode-resource.vscode-cdn.net/Users/mihagazvoda/miniconda3/envs/pymc-dev/lib/python3.11/site-packages/arviz/stats/stats.py:307): FutureWarning: Setting an item of incompatible dtype is deprecated and will raise in a future error of pandas. Value 'log' has dtype incompatible with float64, please explicitly cast to a compatible dtype first.
  df_comp.loc[val] = (
[/Users/mihagazvoda/miniconda3/envs/pymc-dev/lib/python3.11/site-packages/arviz/plots/backends/matplotlib/compareplot.py:87](https://file+.vscode-resource.vscode-cdn.net/Users/mihagazvoda/miniconda3/envs/pymc-dev/lib/python3.11/site-packages/arviz/plots/backends/matplotlib/compareplot.py:87): FutureWarning: Series.__getitem__ treating keys as positions is deprecated. In a future version, integer keys will always be treated as labels (consistent with DataFrame behavior). To access a value by position, use `ser.iloc[pos]`
  scale = comp_df["scale"][0]
```